tests/sync/test_rest_api.py

```
@@ -295,43 +338,77 @@ async def test_denial_end_to_end(self):
+            pass
         except Exception as e:
-            print(f"Error {e}")
+            print(f"Appeals WebSocket error: {e}")
             pass
         finally:
-            await a_communicator.disconnect()
+            if a_communicator:
+                try:
+                    await a_communicator.disconnect()
+                except:
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)